Geometry/HGCalCommonData/plugins/DDHGCalSiliconModule.cc

```diff
@@ -0,0 +1,449 @@
+///////////////////////////////////////////////////////////////////////////////
+// File: DDHGCalSiliconModule.cc
+// Description: Geometry factory class for HGCal (EE and HESil) using
+//              information from the file
+///////////////////////////////////////////////////////////////////////////////
+
+#include "DataFormats/Math/interface/GeantUnits.h"
```

Comment:
It looks like this file uses only angle utilities, not units utilities. If so, it is better to change the include:
```
#include "DataFormats/Math/interface/angle_units.h"
```
Don't bring more dependencies than necessary into the file.